Tasks/ANT/anttask.ts

```diff
@@ -77,6 +77,31 @@ function processAntOutputLine(line) {
     }
 }
 
+function readJavaHomeFromRegistry(jdkVersion: string, arch: string): string {
+    let reg = tl.tool('reg');
```

Comment:
to Eric: I can add that, but I don't think I need to?  If tl doesn't automatically sets the task result (which I don't think it does), error produced by this tool runner is simply ignored.

To Madhuri: I will move the isWindows() check inside this method.